### PR TITLE
Created new cask dbglass

### PR DESCRIPTION
**Pull request by shian15810**
Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/dbglass.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'dbglass' do
+  version 'v0.1.0-beta.6'
+  sha256 'f79f853299f7578f3b30a9df9b2d027ec73ecf933b91f78e287ad2993176f45d'
+
+  url "https://github.com/web-pal/DBGlass/releases/download/#{version}/MAC_OS-X.zip"
```
Comment (miccal) [inline Casks/dbglass.rb]: You need a [comment](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment) above the `url` stanza.

File: Casks/dbglass.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  appcast 'https://github.com/web-pal/dbglass/releases.atom',
+          checkpoint: '8db209134c537d898b9fe48062b87cc65628568055266b3211b959b149877f1d'
+  name 'DBGlass'
+  homepage 'http://dbglass.web-pal.com'
+
+  app "DBGlass-darwin-x64/DBGlass.app", target: 'DBGlass.app'
```
Comment (miccal) [inline Casks/dbglass.rb]: We only use double quotes when interpolation is required - please change the `app` to single quotes.

File: Casks/dbglass.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
```
Comment (miccal) [inline Casks/dbglass.rb]: Please remove the whitespace in this line.

**Review by miccal — CHANGES_REQUESTED** [review]

File: Casks/dbglass.rb (hunk 1 above)
Comment (shian15810) [inline Casks/dbglass.rb]: Thanks for the review, fixed!

File: Casks/dbglass.rb (hunk 2 above)
Comment (shian15810) [inline Casks/dbglass.rb]: Thanks for the review, fixed!

File: Casks/dbglass.rb (hunk 3 above)
Comment (shian15810) [inline Casks/dbglass.rb]: Thanks for the review, fixed!

**Comment by shian15810** [comment]
I've added the comment but the build is still not passing, wonder why?

**Comment by shian15810** [comment]
My fault, should be done by now. Thanks!

**Comment by miccal** [comment]
Thank you.

**Comment by vitorgalvao** [comment]
@shian15810 Your template is incomplete.  The part you left says **additionally** for a reason.  All the problems pointed by @miccal would have been caught (and most even  automatically fixed) by the commands in the first part of the template.

**Comment by shian15810** [comment]
Sorry, I'm new.
What should I do now?

**Comment by vitorgalvao** [comment]
Nothing. Just keep that in mind for the future.

**Comment by shian15810** [comment]
Ok thanks, registered!

**Review by miccal — APPROVED** [review]

**Pull request merged**